src/cmd/burn.rs

```diff
@@ -88,7 +95,8 @@ fn print_txn(
                 ["Amount (HNT)", Hnt::from(txn.amount)],
                 ["Fee (DC)", txn.fee],
```

Comment:
{"data":[{"updated_at":"2022-09-02T13:48:02.830116Z","type":"payment_v2","txn":{"type":"payment_v2","payments":[{"token_type":"hnt","payee":"14Q81QnpYiSBBEVF2Tic31dwh85vwU3QoSYPRqhq3eMJpuqnvbW","memo":"AAAAAAAAAAA=","max":false,"amount":12616033}],"payer":"13NMwZoErb79nevaVbvjr5Aa2yD7HpVgUvMBcfw12tLN8Hfz6ie","nonce":2,"hash":"_bPXVKMEmJGMaDXxd_ysiODIft_VZHpputHoD7hBcNw","fee":35000},"status":"failed","hash":"_bPXVKMEmJGMaDXxd_ysiODIft_VZHpputHoD7hBcNw","failed_reason":"{invalid,insufficient_balance}","created_at":"2022-09-02T13:47:13.608764Z"}]}

Reply:
The link posted above for that commit is broken. Which commit were you referring to? Also what's the purpose of the json output?